src/frontend/components/sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a,b){
```

Comment:
I understanding that this file was created in order to setup jest but it is not needed after the setup has been confirmed to work.